pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -359,6 +359,36 @@ def create_model(df_sets, df_parameters, default={}):
     model.s_B = Set(
         initialize=model.df_sets["TreatmentTechnologies"], doc="Treatment Technologies"
     )
+    model.df_parameters["LLA"] = {
+        **model.df_parameters["PNA"],
+        **model.df_parameters["CNA"],
+        **model.df_parameters["CCA"],
+        **model.df_parameters["NNA"],
+        **model.df_parameters["NCA"],
+        **model.df_parameters["NKA"],
+        **model.df_parameters["NSA"],
+        **model.df_parameters["NRA"],
+        **model.df_parameters["FCA"],
+        **model.df_parameters["RCA"],
+        **model.df_parameters["RNA"],
+        **model.df_parameters["SNA"],
+        **model.df_parameters["RSA"],
+        **model.df_parameters["SCA"],
+    }
+    model.s_LLA = Set(
+        initialize=list(model.df_parameters["LLA"].keys()), doc="Valid Piping Arcs"
+    )
+    model.df_parameters["LLT"] = {
+        **model.df_parameters["PCT"],
+        **model.df_parameters["FCT"],
+        **model.df_parameters["PKT"],
+        **model.df_parameters["CKT"],
+        **model.df_parameters["CST"],
+        **model.df_parameters["CCT"],
+    }
```

Comment:
This list also seems incomplete. Is there a reason that the following types of arcs are missing:

- PST
- PRT
- POT
- CRT
- SCT
- SKT
- RKT

Reply:
Similar to above, these arcs do not yet exist in our input file, so they are not yet a key in df_parameters. I just double checked that all arcs in the input file are indeed included in LLT or LLA

Reply:
Makes sense - similar to the comment I left for LLA, this comment can be disregarded.